scripts/build-glean-metadata.py

```diff
@@ -381,8 +393,8 @@ def get_app_variant_description(app):
         )
 
     # write tag metadata (if any)
-    if app_annotation and app_annotation.get("tags"):
-        tags = [{"name": k, "description": v} for (k, v) in app_annotation["tags"].items()]
+    if app_tags:
```

Comment:
i'm not following this part very well, why do we use `app_tags` as a condition check here instead of `app_annotation and app_annotation.get("tags")`?

Reply:
Because app_tags can now come from two places: both the annotations (e.g. https://mozilla.github.io/glean-annotations/api.json) as well as from probeinfo.tmo (e.g. https://probeinfo.telemetry.mozilla.org/glean/burnham/tags). We merge the two representations above